src/replicache.test.ts

```diff
@@ -436,35 +425,43 @@ test('subscribe with error', async () => {
   cancel();
 });
 
-test.skip('conflicting commits', async () => {
-  // This test does not use pure functions in the mutations. This is of course
-  // not a good practice but it makes testing easier.
-  const ar = resolver();
-  const br = resolver();
-
-  rep = await replicacheForTesting('conflict');
-  const mutA = rep.register('mutA', async (tx, v) => {
-    await tx.put('k', v);
-    await ar.promise;
-  });
-  const mutB = rep.register('mutB', async (tx, v) => {
-    await tx.put('k', v);
-    await br.promise;
-  });
+test('overlapping writes', async () => {
+  async function dbwait(tx: ReadTransaction, dur: number) {
+    // Try to take setTimeout away from me???
+    const t0 = Date.now();
+    while (Date.now() - t0 > dur) {
+      await tx.get('foo');
```

Comment:
Maybe use timerwait in here too? Right now it does as many `get` as it can do in the given time.

Reply:
Cannot use timerwait because the transaction auto-commits when control returns to event loop and no idb ops are pending.

Can just do a busy wait, that might make this more clear.

Reply:
busy wait might get optimized away? This seems fine but maybe add a comment